Gemfile

```diff
@@ -52,3 +51,23 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'font-kit-rails', '~> 1.2.0'
+
+gem 'haml', '~> 5.0.4'
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.7'
+end
+
+group :test do
+  gem 'shoulda-matchers', '~> 3.1'
+end
+
+gem 'factory_bot_rails','~> 4.0'
+
+gem 'rails-controller-testing'
```

Comment:
add lock gem version

Reply:
You can get the latest gem versions from rubygems.org